merge_checks.py

```diff
@@ -85,7 +85,7 @@ def main(head_hash: str, base_ref: str) -> int:
     if incorrect_commit_messages:
         print(
             f"Found invalid commit message, allowed types: {ALLOWED_COMMIT_MESSAGE_TYPES}\n"
-            f"{','.join(incorrect_commit_messages)}\n",
+            f"{chr(10).join(incorrect_commit_messages)}",
```

Comment:
why `chr(10)`?.. I mean it adds nothing to `'\n'` but hurts readability...

Reply:
Yeah, I would go for

```suggestion
            f"{'\n'.join(incorrect_commit_messages)}",
```

Reply:
You suggestion is not possible: `expression fragments inside f-strings cannot include backslashes`.
I tried it before. I can split it up into a separate variable.

Reply:
Oh wow, I didn't know that, then it's fine with me like this.